### ISSUE TITLE
Update the namespace for all examples

### ISSUE THREAD

**Issue by rib** [rust-mobile/rust-android-examples#8]
Currently all the examples are under a `co.realfit` domain-based namespace, except that the `cargo apk` builds will use a default `package = "com.foo.bar"`

We should update this to something like `rust.mobile` and be consistent between Gradle and `cargo apk` builds

**Comment by MarijnS95** [comment]
And we should probably clean out all the commented `cargo-apk` package metadata in `Cargo.toml` that is currently unused, and/or switch to (or at least provide) `xbuild`'s `manifest.yaml`.

**Comment by rib** [comment]
I wonder if you could provide an example xbuild config for na-mainloop in a draft PR? I'd be happy to replace the cargo apk metadata with an xbuild config if you think that would be better?

**Comment by MarijnS95** [comment]
@rib see #9, `cargo-apk` has a sensible default and so has `xbuild`.

**Comment by MarijnS95** [comment]
TLDR: We can support both without providing any configuration for either :grimacing:

**Comment by rib** [comment]
I'll close this now that I've updated most of the examples to now use a `com.github.rust_mobile` namespace
